truenas/webui
# NAS-112308 / 21.10 / Added slide-in modal for the new forms

NAS-112308

To test
1. Check the Localization Form under General Settings with proper slide-in panel
2. Check that the form functions
3. Check the slide-in panel code
4. Is the new code ready to be merged with master in terms of UI/UX consistency
